docs/questions/native-editor/snippets.md

```diff
@@ -150,6 +150,8 @@ you could put the snippet in one query and have the snippet parameter map to a `
 
 If you have multiple snippets containing parameters with the same name, the question using those snippets will only use one instance of the parameter. For example, if `{% raw %}{{snippet: 1}}{% endraw %}` contains parameter `{% raw %}{{var}}{% endraw %}` and `{% raw %}{{snippet: 2}}{% endraw %}` also contains parameter `{% raw %}{{var}}{% endraw %}`, the question will display only one `{% raw %}{{var}}{% endraw %}` parameter and use its value in both snippets.
 
+You can also reuse parameters across different SQL questions by defining them inside a shared snippet. Any question that references the same snippet will share its parameters, so dashboard filters or input values apply consistently across all related queries. For example, if you create a snippet that filters orders by `{{start_date}}` and `{{end_date}}`, you can reuse that snippet in multiple queries, and a single date filter will control them all on a dashboard.
```

Comment:
> a single date filter will control them all...

One filter widget can only connect to one parameter.

```suggestion
To reuse a parameter in multiple SQL questions, define the parameter in a snippet. Any question that references that snippet will share its parameter. For example, if you create a snippet that filters orders by `{% raw %}{{start_date}}{% endraw %}`, you can reuse that snippet in multiple questions. Add those questions to a dashboard, and you can connect a single date filter to each card containing a question  using the snippet with the `{% raw %}{{start_date}}{% endraw %}` parameter.
```

Also: we need raw tags, or this won't render properly on the site.